red/api/flows.js

```diff
@@ -79,5 +80,9 @@ module.exports = {
                 res.status(500).json({error:"unexpected_error", message:err.message});
             });
         }
+       
+        if(cluster.isWorker){
+            
```

Comment:
YES, it's the key to the entire implementation!

...actually no, that was just a left over from a merge.  Good catch!